config/action.d/cloudflare.conf

```diff
@@ -34,7 +36,7 @@ actioncheck =
 #          <time>  unix timestamp of the ban time
 # Values:  CMD
 #
-actionban = curl https://www.cloudflare.com/api_json.html -d 'a=ban' -d 'tkn=<cftoken>' -d 'email=<cfuser>' -d 'key=<ip>'
+actionban = curl -s -o /dev/null https://www.cloudflare.com/api_json.html -d 'a=ban' -d 'tkn=<cfapikey>' -d 'email=<cfemail>' -d 'key=<ip>'
```

Comment:
I believe might simply appear in the terminal where fail2ban-server is running (which is usually none, unless ran manually), thus just nowhere ;)